packages/accounts/src/light-account/accounts/base.ts

```diff
@@ -181,6 +191,16 @@ export async function createLightAccountBase({
     });
   };
 
+  const { signMessage: signMessageV2, signTypedData: signTypedDataV2 } =
```

Comment:
Can we put this in sign with 1271 wrapper function and check the light account version and handle there?

Reply:
I just tried it out and I'm not sure I like it that way. The issue is that then there's branching in two different places: inside `signWith1271Wrapper` and also in the switch statement on the version below. I think I like it more to have distinct functions for each version of the account.

To clarify this a bit, I tried renaming to `signWith1271WrapperV1` to make clear that it should only be used with V1 of accounts.